### DISCUSSION TITLE
Plan for converting to next.js project

### DISCUSSION THREAD

**Discussion by jangbora** [mirumee/saleor-dashboard#2318] [General] [answered]
Do you have a plan for converting react.js project to next.js?

**Comment by peelar** [comment, marked as answer]
Hello @jangbora 👋 

Next.js is a technology that we care much about, so I believe if we ever plan to rebuild Dashboard significantly, it will be under Next.js. However, we have no such plans right now. You will surely be informed when that happens.

**Reply by jangbora** [reply]
Much appreciated!

**Comment by timuric** [comment]
@jangbora please refer to this thread for hosting dashboard on vercel with next https://github.com/saleor/saleor-dashboard/discussions/2446#discussioncomment-3957864
